### ISSUE TITLE
ALL: H2 in Main Navigation appears in document before H1

### ISSUE THREAD

**Issue by imcbride** [middlebury/midd-frontend#539]
https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=64768883914&impmd=ERAQQMIEOUHRRKBGATMU&decision=dismissed+falsePositive+none#/sia-r61/failed/ZWNDUktvRHo=;NjgxNGViNjc=

https://github.com/middlebury/midd-frontend/blob/master/src/templates/partials/school-header.twig#L34

**Comment by poojagunturu96** [comment]
Issue still exists on several other pages as listed in this report: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?ruleId=61&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2. 

There are several instances of an `<h2 class="sr-only">` tag being used for labelling elements in the frontend code which need to be reviewed: https://github.com/search?q=repo%3Amiddlebury%2Fmidd-frontend%20h2%20class%3D%22sr-only%22&type=code.
